mmv1/third_party/terraform/resources/resource_logging_bucket_config.go

```diff
@@ -155,6 +197,18 @@ func resourceLoggingBucketConfigCreate(d *schema.ResourceData, meta interface{},
 	obj["retentionDays"] = d.Get("retention_days")
 	obj["locked"] = d.Get("locked")
 
+	cmekSettingsData := d.Get("cmek_settings").([]interface{})
+	if len(cmekSettingsData) > 0 && cmekSettingsData[0] != nil {
+		cmekSettings := cmekSettingsData[0].(map[string]interface{})
+
+		obj["cmekSettings"] = map[string]interface{}{
+			"name":              cmekSettings["name"],
+			"kmsKeyName":        cmekSettings["kms_key_name"],
+			"kmsKeyVersionName": cmekSettings["kms_key_version_name"],
+			"serviceAccountId":  cmekSettings["service_account_id"],
+		}
+	}
+
```

Comment:
I have a nit and a question related to this block:
- Nit: I think this could be an `expand*` function, which might be a little more clear/consistent with other resources, but it probably isn't necessary here if you'd prefer to keep it directly in the "create".
- I think part of the benefit of the `expand*` functions is that they are typically reused for both "create" and "update". Do you know why `resourceLoggingBucketConfigUpdate()` does not need the same behavior? I'm assuming the implementation you have is working because of the test.

Reply:
Yeah, that is a good point.  I moved the code block to an expand* function.